### ISSUE TITLE
Implement GetLayerNames()

### ISSUE THREAD

**Issue by Sauraus** [hybridgroup/gocv#388]
I have the need to do some heavy YOLO work and as part of that need the function `GetLayerNames()` to extract the output layers like its done in python:

```
	# determine only the *output* layer names that we need from YOLO
	ln = net.getLayerNames()
	ln = [ln[i[0] - 1] for i in net.getUnconnectedOutLayers()]
```

Didn't see this on the roadmap so I am wondering if a PR will be accepted for this.

**Comment by Sauraus** [comment]
That's also an option yes :) I will take a look at implementing that method instead, or might just do both time permitting.

**Comment by deadprogram** [comment]
A PR that added would be gladly accepted, @Sauraus

**Comment by deadprogram** [comment]
This was addressed by #398 so now closing. Thank you!

**Comment by hamzsabe** [comment]
YOLOv3 has 3 output layers (82, 94 and 106) as the figure shows.

getLayerNames(): Get the name of all layers of the network.

getUnconnectedOutLayers(): Get the index of the output layers.

These two functions are used for getting the output layers (82,94,106). I prefer to use the following code for simplicity:

import cv2 
import numpy as np 
import time 
#Loading Yolo 
net = cv2.dnn.readNet("yolov3.weights", "yolov3.cfg") 
classes = [] 
with open("coco.names", "r") as f: 
        classes = [line.strip() for line in f.readlines()] 
outputlayers = net.getUnconnectedOutLayersNames()
